code/lighting/lighting.cpp

```diff
@@ -1251,21 +1251,21 @@ bool scene_lights::setLights(const light_indexing_info *info)
 		return false;
 	}
 
-	int index_start = info->index_start;
-	int num_lights = info->num_lights;
+	auto index_start = info->index_start;
+	auto num_lights = info->num_lights;
 
 	// check if there are any lights to actually set
-	if ( num_lights <= 0 || index_start < 0 ) {
+	if ( num_lights <= 0 ) {
```

Comment:
The condition `index_start < 0` could never be true in the original code since it only was set to 0 or the size of a vector so I just removed the condition since it caused compilation warnings.